cmd/jobs.go

```diff
@@ -80,6 +135,10 @@ func init() {
 	jobsRunCmd.Flags().BoolVar(&flagJobPrintLogs, "logs", false, "print jobs log in stdout")
 	jobsRunCmd.Flags().BoolVar(&flagJobPrintLogsVerbose, "logs-verbose", false, "verbose logging (with --logs flag)")
 
+	jobsPurgeCmd.Flags().StringSliceVar(&flagJobWorkers, "workers", nil, "worker types to iterate over (all workers by default)")
+	jobsPurgeCmd.Flags().IntVar(&flagJobsPurgeMonth, "months", 2, "number of months to look for")
```

Comment:
I'd be better to have this default of `2` in a single place: maybe something like `jobs.DefaultNumberOfMonthsToKeep`.